src/app/components/MostRead/index.test.tsx

```diff
@@ -258,16 +258,16 @@ describe('MostRead', () => {
       process.env.SIMORGH_APP_ENV = originalEnvironment;
     });
 
-    it('should not render a most read component when the environment is live', () => {
+    it('should render a most read component when the environment is live', () => {
```

Comment:
Ah I didn't realise that. Yes, ideally it should also be deleted, it was meant to be temporary so that we could continue to develop on live without accidentally releasing the new feature 😸